theory/Theory.md

```diff
@@ -0,0 +1,13 @@
+#### Single regex that matches either of these: `antelope rocks out`, `antelopes rock out`
+=> `([a-zA-Z]+?)(s\b|\b)`
+
+#### Regex that matches either of: `goat`, `moat`, but not: `boat`
+=> `[^b]oat`
```

Comment:
This also works but isn't comprehensive enough. The word `coat` will match. Try something like this:

```
/(g|m)oat/
```